src/App.jsx

```diff
@@ -1,15 +1,25 @@
-import React from 'react';
-import { useFeaturedBanners } from './utils/hooks/useFeaturedBanners';
+import React, { useState } from 'react';
 import { Header, Home, Footer } from './components';
+import { Button, ButtonWrapper } from './App.style'
+import ProductList from './components/ProductList';
 
 function App() {
-  const { data, isLoading } = useFeaturedBanners();
-  console.log(data, isLoading);
-
+  const [pageShowed, setPageShowed] = useState('Home');
```

Comment:
what other values can be set on pageShowed? if is only one for one if else, maybe a boolean is better approach. Just minor detail.